SMLHelper/Json/Converters/ColorConverter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+using System;
+using UnityEngine;
+
+namespace SMLHelper.Json.Converters
+{
+    internal class ColorConverter : JsonConverter
+    {
+        /// <summary>
+        /// A method that determines when this converter should process.
+        /// </summary>
+        /// <param name="objectType">the current object type</param>
+        /// <returns></returns>
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(Color);
+        }
+
+        /// <summary>
+        /// A method that tells Newtonsoft how to Serialize the current object.
+        /// </summary>
+        /// <param name="writer"></param>
+        /// <param name="value"></param>
+        /// <param name="serializer"></param>
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            Color vector3 = (Color)value;
+            serializer.Serialize(writer, (ColorJson)vector3);
```

Comment:
This will bug me to high heaven if it is not fixed
```suggestion
            Color color = (Color)value;
            serializer.Serialize(writer, (ColorJson)color);
```

Reply:
Lol WHOOPS, you can probably guess where I got that ;)

Reply:
Happy to commit the suggestion?